apps/ui/src/composables/useSpaceSettings.ts

```diff
@@ -418,15 +418,18 @@ export function useSpaceSettings(space: Ref<Space>) {
   }
 
   function getInitialVotingProperties(space: Space) {
+    const validPrivacyTypes = ['shutter', 'any'];
     const spaceVoteType = space.additionalRawData?.voting.type;
-    const privacyValue = space.additionalRawData?.voting.privacy;
+    const privacyValue = space.additionalRawData?.voting.privacy || 'none';
```

Comment:
```suggestion
    const privacyValue = space.privacy;
```
It's now part of space type and we don't need to add fallback for it.

Reply:
> It's now part of space type and we don't need to add fallback for it.

the fallback from empty string to `none` is still needed, as the hub still return `""`, and not `"none"`.

Reply:
https://github.com/snapshot-labs/sx-monorepo/blob/eea6cd81f4961d03cb031f20ebe37ae2b9ffc032/apps/ui/src/networks/offchain/api/index.ts#L205

`"" || "none" === "none"`

Reply:
You're correct, though we're still in the formatSpace function

Reply:
Not sure what you mean. That means that `Space` will have defined `privacy` and it won't be empty string. I also suggested we use `space.privacy` here so it should be the same thing 🤔

Reply:
You’re correct, my mind was somewhere else